app/uk/gov/hmrc/gform/sharedmodel/form/Seed.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.sharedmodel.form
+
+import java.util.UUID
+
+import play.api.libs.json._
+import uk.gov.hmrc.gform.sharedmodel.ValueClassFormat
+
+case class Seed(value: String)
```

Comment:
I don't understand what this class is giving us. Why don't we just instantiate the Form a SubmissionReference instance, rather than giving it this Seed thing & then deriving a SubmissionReference from it?

If this has been discussed & everybody is happy with it, feel free to ignore this comment.